### ISSUE TITLE
INPUT: Mulitup.org

### ISSUE THREAD

**Issue by MachineGunnur** [MachineGunnur/GOG-Games#47]
I been thinking and reading. How about use file mirroring site like: https://multiup.org ?

Let list PROS/CONS:

**PROS**
- support up to 10 file hoster
- Jdownloader works with it
- API support

**CONS**
- have to wait for service to mirror file
- speed of upload from our server to their (NOT TESTED, may be quick.)

Any else want to make inputs here? Thanks!

**Comment by MiiKe999** [comment]
Please don't remove Google Drive, it's very reliable and fast anytime. Apart from that you can add multiple mirroring sites if you want, but don't remove Google Drive.

**Comment by gahrensc** [comment]
I agree with MiiKe, Drive is very reliable and fast. It is possible to do the work around much easier using Drive API. Anyone who cares enough to complain about can try make their own script, otherwise they do not have right to complain. I think multiup support Drive but it would be useful to put drive links directly on site for people who know how to use them, can probably get drive links from multiup API when uploading. Thanks.

**Comment by fopsc** [comment]
you will probably need to return to zipping the content, and find a split size that works with every host. This is a PRO, because we can mix and match the sources. btw I suggest 7zip.

Please provide a test link once you start experimenting with it, the same kind of link that you would put on the site. So we can provide feedback.

**Comment by nadoeli** [comment]
I'd vote for multiup as long as it supports zippyshare, I really miss this hosting, it required literally 2 clicks to download from it (copy all links + confirm in jdownloader)

**Comment by kevinj93** [comment]
More filehosts = less complaining from users and more convenient for everyone. I think the 500MB parts would work with most of the well-known filehosts without any problem. My preferred filehosts are:  megaup.net, mega.nz, uptobox, zippyshare and openload.co. The others are not very free-user friendly

**Comment by MachineGunnur** [comment]
I did some test with Multi up:
megaup doesn't work nor openload..
mega.nz requires account - dont have and free is low limit
google drive option don't work for team drive which we have.

**Comment by MachineGunnur** [comment]
> 
> 
> Please don't remove Google Drive, it's very reliable and fast anytime. Apart from that you can add multiple mirroring sites if you want, but don't remove Google Drive.

we can only use one hoster. the server payment would be even more monthly if we uploading too multiple site.

**Comment by kevinj93** [comment]
uptobox and 1fichier are very good. 1fichier has a long data retention rate and their premium subscription is like ~1.5 $ per month and 12$ per year. So maybe uploading to uptobox/1fichier/gdrive (if supported)/zippyshare would be more than enough for most of the users

**Comment by MachineGunnur** [comment]
i have mulitup.org upload written - very simple. i think do a test for maybe a week? the goole quota is annoy... so i think will do a poll and be mulitup  vs zippy.

also long term - google can get taken away then all things LOCKED!

**Comment by kevinj93** [comment]
You can also give MirrorAce a try, they also have API upload support, and it supports lots of free (no account) filehosts as well. Personally I have had very good experiences with MirrorAce

**Comment by MachineGunnur** [comment]
OK will test.

**Comment by MachineGunnur** [comment]
openload broken too, think something is wrong with them. in my testing it was slow, stalled, not preforming good.

also, mirrorace triggers annoying captcha with jdownload

**Comment by sezglt** [comment]
Yeah, MirrorAce shows Capcha from Tor! Could try alternatives to: [Go4up](https://go4up.com/misc/apidoc), [Multifetchmirror](https://multifilemirror.com/page/api), [Mirrored](https://www.mirrored.to/p/api-doc)

**Comment by MachineGunnur** [comment]
I am just revert back to zippy. new uploads will use zippy. Can;t upload everything for few week until bandwidth reset. So have google links until then. I don't want to be brutal, but I pay for storage, hosting and put in time when I have to update and code stuff. What I do to site is up to me, I say.  Thank you for everyone that gives input. It appreciated!

**Comment by MachineGunnur** [comment]
some more infos here on current situtaion: https://github.com/MachineGunnur/GOG-Games/issues/58#issuecomment-516057188

**Comment by MachineGunnur** [comment]
did lots of testing.. the reliability of mulitup mirroring is a crapshoot!

**Comment by MachineGunnur** [comment]
here is what i talk about:
https://multiup.org/en/mirror/b69d3c06fce3c430c697f3796594427f/game-fallout_classic.rar

not all hosters are there. they say they are but if you click it redirects to ads....

**Comment by MachineGunnur** [comment]
i report to MU about problem. happens many times via API upload..

**Comment by MachineGunnur** [comment]
guys, i speak to soon! all links there. now. takes  awhile even though doesn't say uploading on MU. very confused, but works ;)

**Comment by MachineGunnur** [comment]
Poll on which hosters! vote here: https://xoyondo.com/ap/KZncq2ZWk3D4zE5

some are not there, like megaup, as do not work at all.

**Comment by galneon** [comment]
When GOD used Multiup, there was always a chance some files would only make it to 8-9 hosters. However, it's not hard to put together a complete set from two hosters in that event.

**Comment by MachineGunnur** [comment]
every solution has their downsides.

GOOGLE: quota
ZIPPY: EU block
MULTIUP: chance of fail on certain host

**Comment by totemor** [comment]
Could grab URLs to made non-existent for users, any way? Show Zippyshare, Openload or GDrive as if are yours. Not against but Multiup is kinda intrusive with uBlock, popups on click, many pages are [flagged](https://transparencyreport.google.com/safe-browsing/search?url=https%3A%2F%2Fmultiup.org%2Fredirect-to-host%2Fzippyshare.com%2Fde7de269692cd0fac875615be7e98c13%2F3) by Google Safe Browsing, shows Captcha.

**Comment by MachineGunnur** [comment]
@totemor use jdownloader 2........

**Comment by MachineGunnur** [comment]
also, i think 1fichier is broken on mulitup. can anyways test? don't work here.

**Comment by MachineGunnur** [comment]
oh, reliability on site is 9% :-D ... send notice to MU about it broken.

**Comment by totemor** [comment]
This "no ads" in Cups is an weird ad by itself, Jdownload2 is Bundleware. Plus more apps bring more risks, so no, thanks. If "easy" to automate, why struggle to give the links up? And you know, it's so annoying to have Captcha from Tor, Zippyshare or Google Drive never have had. For many months Zippyshare was the best and still is.

I was known as Guaper or Quiapho. MachineJunkie was flagged by Github and can't do anything about. To cups me is totally unnecessary, I am checking the repo's issues anyway. Just hope Google Drive's acc won't be deactivated and old links will technically continue to work even after removal from the site as they actually continue to do. Honestly, thank you for! Sacred 2 Gold and Descent are fixed, now can get data almost directly. To trust only GG's source, GG's server and Google Drive's acc is less than a chain of GG's source, GG's server, Multiup's Infrastructure and a Cloud combined.

Do you remember Tox's answer about Jdownloader2? PyLoad! When possible, I prefer to avoid what's unnecessary, this things are risky. I could not question even such Multiup if GG did provide BLAKE2 for Goodies ([not CRC32](https://github.com/MachineGunnur/GOG-Games/issues)): Zip has no signatures, no protection. Multiup's poll shows people like Zippyshare, so why not drop experimenting and reverse to what worked well???

Don't remember if wrote but you can go to RIN and on the page 480 of GOG Collection, I have a lot of scrapped Zippyshare links from GG, can use to recover around 3 TB for free, it's from July 12. In addition I did force the Wayback Machine to save the whole GG on July 30. Anyway, better to setup a task on cron for backups, if was the case, could easily come back to Zippyshare.

I should insist to claim that Zippyshare is not available for 700 Million or the EU is Very Misleading, because the number is far less than 100 and those ones can use a free VPN and have no issues whatsoever: 3 Countries is a part if the EU but not the whole EU. I don't see any reason to avoid Zippyshare or to not mix it with Google Drive (for Goodies); Multiup feels crappy, many things are broken.
